app/src/main/java/com/example/playlistmaker/mvvm/library/ui/activity/PagerAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.playlistmaker.mvvm.library.ui.activity
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.Lifecycle
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class PagerAdapter(fragmentManager: FragmentManager, lifecycle: Lifecycle) :
+    FragmentStateAdapter(fragmentManager, lifecycle) {
+
+    override fun createFragment(position: Int): Fragment {
+        return when (position) {
+            0 -> LikedTracksFragment.newInstance()
+            1 -> PlaylistFragment.newInstance()
+            else -> LikedTracksFragment.newInstance()
```

Comment:
Здесь можно сократить:
```suggestion
            0 -> LikedTracksFragment.newInstance()
            else -> PlaylistFragment.newInstance()
```